### ISSUE TITLE
Visual distinction between directory empty and still being loaded

### ISSUE THREAD

**Issue by mherrmann** [fman-community/fman#535]
At the moment, the following two situations look the same in fman:

 * The current directory is empty
 * fman is still loading the current directory.

The goal of this issue is to give some visual indication which of the two is the case. Possible ideas:

 1. Show a spinning "loading" symbol to show that files are still being read.
 2. In empty directories, display the text "This directory is empty" (or similar).

**Comment by IgorKrupenja** [comment]
Yep, this would be especially useful when browsing directories mounted over SSHFS.
